ecosystem/sep-0010.md

```diff
@@ -22,18 +22,23 @@ The authentication flow is as follows:
 
 1. The client obtains a unique [`challenge`](#challenge), which is represented as specially formed Stellar transaction
 1. The client verifies that the transaction has an invalid sequence number 0.  This is extremely important to ensure the transaction isn't malicious.
-1. The client signs the transaction using the secret key for the user's Stellar account
+1. The client signs the transaction using the secret key(s) of signers for the user's Stellar account
 1. The client submits the signed challenge back to the server using [`token`](#token) endpoint
+1. The server gets the signers of the user's account
+1. The server verifies the signatures
+1. The server verifies the weight provided by the signers meets the required threshold(s), if any
```

Comment:
What is meaning of `required thresholds` in this context?

Reply:
Any threshold the implementer wants the client to meet on the account. In many cases and for simplicity an implementer should probably just confirm that the signatures meet all thresholds (low, med, high) on an account to ensure the user completely holds an account in the same way they do if they have the default master key set as a signer.

An implementer can change this however they need for their specific application, requiring any lower threshold or no threshold. See this section:
https://github.com/stellar/stellar-protocol/blob/d7f11aba047a59989d0323981ec760360748b20a/ecosystem/sep-0010.md#L141-L145